sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/RoslynProfileExporter.java

```diff
@@ -217,6 +229,15 @@ private String analysisSettings(boolean includeSettings, boolean ignoreHeaderCom
       appendLine(sb, "  </Settings>");
     }
 
+    if (includeSettings) {
```

Comment:
I could have avoided this change, but until we remove this file, it's best to be as "up to date" as possible, to avoid confusion